addons/spreadsheet/utils/validate_data.py

```diff
@@ -176,7 +177,7 @@ def fields_in_spreadsheet(data):
     """return all fields, grouped by model, used in the spreadsheet"""
     all_fields = chain(
         extract_fields(list_fields, data.get("lists", dict()).values()).items(),
-        extract_fields(pivot_fields, data.get("pivots", dict()).values()).items(),
+        extract_fields(pivot_fields, (pivot for pivot in data.get("pivots", dict()).values() if pivot.get("type", "ODOO") == "ODOO")).items(),
```

Comment:
i'd put the pivots list in a separate varaible for lisibility. (not sure if we have line lenght guidelines for python ?)